egs/wsj/s5/utils/reverse_lm.sh

```diff
@@ -38,25 +38,13 @@ mkdir -p $outdir
 for f in phones.txt words.txt L.fst L_disambig.fst phones/; do
   cp -r $langdir/$f $outdir
```

Comment:
The reverse_lm script is used only in aurora4/s5 and sprakbanken/s5. The latter is free, so I can probably pull a real grammar. This should be thorough enough.

Another interesting question is should we try to implement and maintain unit tests for the utility scripts (and executable tools by extension). Sounds nasty, but a quick search reveals a few bash unit-testing frameworks. The implementation is for another day indeed, as there is just one of me, unfortunately. But as a point to be in--does it make sense?